src/main/kotlin/it/pagopa/wallet/eventdispatcher/domain/WalletEvent.kt

```diff
@@ -7,9 +7,16 @@ sealed interface WalletEvent {
     val creationDate: OffsetDateTime
 }
 
-data class WalletUsed(
+data class WalletUsedEvent(
     override val eventId: String,
     override val creationDate: OffsetDateTime,
     val walletId: String,
     val clientId: String
 ) : WalletEvent
+
+/** Wallet created event. */
+data class WalletCreatedEvent(
```

Comment:
Cause on Wallet service side is called `WalletExpiredEvent`, what about using the same name?

Reply:
Let's have the same _eCommerce_ convention also here. Wallet service, once create a new wallet will send a `WalletCreatedEvent` since the wallet has been created in that moment with a visibility timeout set as per process and wallet-event-dispatcher process it checking if wallet have reached a final state in the meantime